### PR TITLE
Tuos joshua giles/subtract

### PR DESCRIPTION
**Pull request by TUOS-JoshuaGiles**

### CODE REVIEW

File: pythoncalculator/subtract.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def subtract(x, y):
+    return y - y
```
Comment (RikY-PinKmaN) [inline pythoncalculator/subtract.py:2]:
```suggestion
    return x - y
```

**Review by RikY-PinKmaN — CHANGES_REQUESTED** [review]
I think this should work

**Review by TUOS-JoshuaGiles — COMMENTED** [review]
Looks good

**Review by RikY-PinKmaN — APPROVED** [review]
Looks good @TUOS-JoshuaGiles

**Pull request merged**